core-services/prow/02_config/_config.yaml

```diff
@@ -133,6 +133,32 @@ deck:
         slug: openshift-team-operator-runtime
       - org: openshift
         slug: openshift-team-operator-ecosystem
+  - repo: opendatahub-io/notebooks
+    rerun_auth_configs:
+      github_team_slugs:
+      - org: opendatahub-io
+        slug: notebook-devs
+      github_users:
+      - atheo89
+      - caponetto
+      - daniellutz
+      - harshad16
+      - jiridanek
+      - jstourac
+      - paulovmr
+  - repo: red-hat-data-services/notebooks
+    rerun_auth_configs:
+      github_team_slugs:
+      - org: red-hat-data-services
+        slug: notebook-admins
+      github_users:
+      - atheo89
+      - caponetto
+      - daniellutz
+      - harshad16
+      - jiridanek
+      - jstourac
+      - paulovmr
```

Comment:
@andyatmiami , @dibryant , maybe add them?

Reply:
explained in person, perms are also granted through a github group, which includes everyone